### ISSUE TITLE
Show LEDs on hardware does not match the old behavior

### ISSUE THREAD

**Issue by riknoll** [microsoft/pxt-microbit#2908]
In the latest beta, calling showLeds clears the screen after the interval has passed. The old behavior was to leave the last displayed image up on the screen.

Here's a MakeCode program that demonstrates it:

```ts
input.onButtonPressed(Button.A, function() {
    basic.showLeds(`
    . . . . .
    . # . # .
    . . . . .
    # . . . #
    . # # # .
    `)
})
```

This only shows up on the hardware, not in the sim

**Comment by riknoll** [comment]
@finneyj FYI

**Comment by pelikhan** [comment]
I'll investigate to see if this was my change.

**Comment by pelikhan** [comment]
Regression in microbit-dal https://github.com/lancaster-university/microbit-dal/issues/463

**Comment by finneyj** [comment]
arrgh. Though I'd nailed these - sorry guys.

To clarify, for consistency in MakeCode, all print() methods (be it char, string or image) should *not* clear the screen for length==1 operations?

**Comment by finneyj** [comment]
clarification - even when a duration is explicitly set?

**Comment by finneyj** [comment]
Apologies. Fixed in  microbit-dal v2.2.0-rc5

**Comment by pelikhan** [comment]
No clearing when it does not scroll.
